src/components/slider/slider.styles.ts

```diff
@@ -0,0 +1,119 @@
+import { useTheme } from 'native-base';
+import { StyleSheet, TextStyle } from 'react-native';
+
+export const useSliderStyles = () => {
+  const theme = useTheme();
+
+  return StyleSheet.create({
+    container: {
+      flexDirection: 'row',
+      gap: theme.space['0'],
+      justifyContent: 'center',
+      alignItems: 'center',
+      width: '100%',
+    },
+    horizontalTrack: {
+      width: '100%',
+      marginTop: theme.space[2],
+      height: theme.sizes['2'],
+      borderRadius: theme.radii.full,
+      justifyContent: 'center',
+      backgroundColor: theme.colors.primary['50'],
+    },
+    filledTrack: {
+      position: 'absolute',
+      borderRadius: theme.radii.full,
+      backgroundColor: theme.colors.primary['500'],
+      height: theme.sizes['2'],
+    },
+    handle: {
+      position: 'absolute',
+      width: theme.sizes['5'],
+      height: theme.sizes['5'],
+      backgroundColor: theme.colors.primary['500'],
+      borderRadius: theme.radii.full,
+      borderWidth: parseInt(theme.borderWidths['4'], 10),
+      borderColor: theme.colors.white,
+    },
+    valueBubbleWrapper: {
```

Comment:
Can we take a hard look at the names of the components used here?

The goal is they should be extremely easy to relate to when one looks at the slider. Mentally, I can break slider into few pieces - valueLabel, sliderLine, sliderHandle and so forth. `valueBubbleWrapper` is not easy to relate to.